networkx/convert.py

```diff
@@ -248,12 +248,91 @@ def to_dict_of_dicts(G, nodelist=None, edge_data=None):
     nodelist : list
        Use only nodes specified in nodelist
 
-    edge_data : list, optional
-       If provided,  the value of the dictionary will be
-       set to edge_data for all edges.  This is useful to make
-       an adjacency matrix type representation with 1 as the edge data.
-       If edgedata is None, the edgedata in G is used to fill the values.
-       If G is a multigraph, the edgedata is a dict for each pair (u,v).
+    edge_data : singleton, optional
+       If provided, the value of the dictionary will be set to `edge_data` for
```

Comment:
Unlike markdown, I believe rst wants inline code to be surrounded by double backticks: e.g.

```
``edge_data``
```

instead of 

```
`edge_data`
```

Not sure how sphinx handles this though. It might be fine.

Reply:
According to the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts), parameters should still be enclosed in single backticks (default role), not double-backticks (inline literal).